.cirrus.yml

```diff
@@ -17,6 +17,7 @@ task:
   build_script:
     - . $HOME/.cargo/env
     - cargo build --all-targets --verbose
+    - cargo build --all-targets --verbose --no-default-features
```

Comment:
You don't need this -- it is already handled by the Azure Pipelines CI. Cirrus is just there to test on FreeBSD.